### DISCUSSION TITLE
Approach to multi-level isometric grid

### DISCUSSION THREAD

**Discussion by JernejHabjan** [rexrainbow/phaser3-rex-notes#315] [Q&A]
I've seen the [isometric grid example](https://github.com/rexrainbow/phaser3-rex-notes/discussions/245),
but it's only 1 level.
Would it make sense to introduce lvl 2 height the same way with the `board` or how could one tackle this?
![image](https://user-images.githubusercontent.com/21958321/193892056-4a0ea17f-e516-46e9-9335-1cacd07e4c5e.png)
![image](https://user-images.githubusercontent.com/21958321/193892641-89dbcbc2-7c15-4c7b-8242-1bd5902aa45c.png)

Or to even introduce sloped terrain:
![image](https://user-images.githubusercontent.com/21958321/193892579-9822e0dc-f669-4334-a81f-1cd92c88a1a3.png)

**Comment by rexrainbow** [comment]
This feature is so cool, but I don't have plan to add it into board plugin recently.
